pkgs/development/python-modules/rtfde/default.nix

```
@@ -39,8 +41,8 @@ buildPythonPackage rec {
   pythonImportsCheck = [ "RTFDE" ];
 
   disabledTests = [
-    # Content mismatch
-    "test_bin_data_captured"
+    # Malformed encapsulated RTF discovered
+    "test_encoded_bytes_stay_encoded_character"
   ];
 
   meta = with lib; {
```

```suggestion
  meta = {
```
Get rid of that too if you don't mind.

---

Fixed!